app/controllers/repositories_controller.rb

```diff
@@ -1,31 +1,6 @@
 class RepositoriesController < ApplicationController
-  before_action :require_author_authentication, only: %i[new create]
```

Comment:
the update should need :require_author_authentication check?

Reply:
Correct, it's been fixed.